### ISSUE TITLE
FR: countdown widget / slider replacement (dimmer control)

### ISSUE THREAD

**Issue by M0ebiu5** [hobbyquaker/node-red-contrib-polymer#23]
This widget should visualize a countdown as clearly as possible - so one can see it from the other side of the room :) 

It could look like gauge 19 or 15:
![clipboard01](https://cloud.githubusercontent.com/assets/9152698/23673562/0a7314fa-0373-11e7-92a3-31b7d0865043.jpg)

Gauge15 
should be full on countdown start and then loose "water/sand/light". 
Also it should optionally fade to a custom color during progress. Number maybe bigger and centered.

Gauge19 
from a full to empty

Both should have optional color fading between start and end point and options if / how the countdown can be started, canceled or reseted. Possible actions on click: start, pause, reset, restart. The interval can be fixed or configurable (slide or press up/down).

I think this widgets would also be nice, intuitive dimmer controls. Sliding up or pressing the upper half to brighten and vice versa.

**Comment by hobbyquaker** [comment]
alternative sliders - yes, I've thought about this too.
But the countdown-widget - hmmm... do we really need this or can't we just use a common gauge-like widget and create the countdown out of node-red?

**Comment by M0ebiu5** [comment]
Yes, but handling the timer states makes the flow unnecessary confusing and bulky.
